src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/XDocExample1.java

```diff
@@ -0,0 +1,15 @@
+/*
+AbstractClassName
+format = (default)^Abstract.+$
+ignoreModifier = (default)false
+ignoreName = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+
+abstract class AbstractFirstClass {} // OK
+abstract class SecondClass {} // violation
+class AbstractThirdClass {} // violation
```

Comment:
In the site examples, these have a following such as `violation, must be declared 'abstract'`
https://checkstyle.sourceforge.io/config_naming.html#AbstractClassName_Examples

Do we keep those or do we aim to keep examples as short as possible?

@romani @nrmancuso @rnveach

Reply:
I think a very concise explanation is good to help the reader zero in on what’s important. Obviously some messages can be shortened or improved in some way if reasonable, but if we are automating this process I would just keep them as is for now.

Reply:
We can apply same logic as we did for filling Inputs with trailing comments. Do  only `//violation` at first.
As separate activity, fill with violation message if check has several violations and it match our expectations on how it looks, and it isn't weird in looks.

Reply:
Ones we did 20 check update , we will see how it it looks like if we add violations. 
It might be too wide, but we try to be shorter horizontally

Reply:
Resolved